src/jit/hwintrinsicArm64.h

```diff
@@ -18,16 +18,22 @@ struct HWIntrinsicInfo
         IsSupported, // The IsSupported property will use this form
         Unsupported, // Any intrisic which is unsupported and must throw PlatformNotSupportException will use this form
         // Non SIMD forms
-        UnaryOp, // Non SIMD intrinsics which take a single argument
-        CrcOp,   // Crc intrinsics.
+        UnaryOp,      // Non SIMD intrinsics which take a single argument
+        CrcOp,        // Crc intrinsics.
+        Sha1RotateOp, // instrisics for SHA1 Hash Rotate operations. Takes hash index unsigned int and returns unsigned
+                      // int
         // SIMD common forms
-        SimdBinaryOp, // SIMD intrinsics which take two vector operands and return a vector
-        SimdUnaryOp,  // SIMD intrinsics which take one vector operand and return a vector
+        SimdBinaryOp,         // SIMD intrinsics which take two vector operands and return a vector
+        SimdUnaryOp,          // SIMD intrinsics which take one vector operand and return a vector
+        SimdBinaryOverlapOp,  // Same as SimdBinaryOp , with destination vector same as first source vector
```

Comment:
Would it make sense to describe these as RMW (read-modify-write) Ops? That's the term used for xarch operations that overwrite their first source, and to me "Overlap" generally implies a more complex kind of operation that does a partial (re)write or something.

Reply:
@CarolEidt I guess renaming them to "RMW" from "Overlap" should be OK. Will do that.